### PR TITLE
VTCCA-3651: Scala 2.12 and SBT 1.3

### PR DESCRIPTION
**Pull request by PeterPerhac**

### CODE REVIEW

File: app/services/RegistrationService.scala [hunk 1]
```diff
@@ -71,7 +71,7 @@ class RegistrationService @Inject()(
         option: Option[DetailedIndividualAccount],
         addressId: Long,
         groupAccount: Option[GroupAccount],
-        affinityGroupOpt: Option[AffinityGroup] = None)(
+        affinityGroupOpt: Option[AffinityGroup])(
```
Comment (darrell-plant) [inline app/services/RegistrationService.scala:74]: just double-checking you are sure about removing the default (None)?

**Comment by darrell-plant** [comment]
LGTM 👍

File: app/services/RegistrationService.scala (hunk 1 above)
Comment (PeterPerhac) [reply app/services/RegistrationService.scala:74]: of course, scala 2.12 can spot when the default value is never used. all callers explicitly provide this argument. neat

**Comment by habeeb01** [comment]
👍 LGTM

**Pull request merged**